### ISSUE TITLE
SSL handshake hook to support dynamic SNI

### ISSUE THREAD

**Issue by pvdvreede** [nginx-clojure/nginx-clojure#110]
Hi there,

Great project, awesome to be able to use Clojure to hook into nginx.

Is it possible to add a hook during the SSL handshake so that I can have a dynamic server block but still support SNI? Basically like `ssl_certificate_by_lua` in open resty.

Thanks.

**Comment by xfeep** [comment]
@pvdvreede No. It will be in the TODO list but it will get a low priority.
